manifests/grafana-dashboardDefinitions.yaml

```diff
@@ -22243,7 +22243,7 @@ items:
                           "timeShift": null,
                           "title": "CPU Utilisation",
                           "tooltip": {
-                              "shared": false,
+                              "shared": true,
```

Comment:
I suspect those changes are here because we are no longer importing all dashboards into one global variable and they don't share one config.